addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -461,6 +461,10 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
                     console.log(log);
                 });
                 console.log(document.body.parentElement.outerHTML);
+                // Also log iframes HTML
+                document.body.querySelectorAll('iframe').forEach(iframeEl => {
+                    console.log(iframeEl.contentDocument.body.parentElement.outerHTML);
+                });
```

Comment:
I was expecting something better here where the empty iframe tag that is logged with the page body is filled by its actual content... :grimacing: Just adding an extra log will make life harder for people trying to copy paste the page body log (while copy / pasting with iframes filled with their content allows for better debugging + the browser just ignores the content if pasted via the editor tools).

Also since we introduce the `o_iframe` class for the iframes that needs to be considered for mutation, let's also only consider those for the logs. It should also make the regex-replacing I mentioned above more robust I guess :shrug: